reference/sisdata.v1.yaml

```diff
@@ -173,14 +230,22 @@ components:
         - id
         - name
         - school
-        - schoolYear
-    teacherOfGroup:
+    groupReference:
       type: object
+      x-examples: {}
       x-tags:
-        - teacher
         - group
       description: ''
-      title: teacherOfGroup
+      properties:
+        id:
+          type: string
+        name:
+          type: string
+      required:
+        - id
+        - name
+    teacherOfClass:
```

Comment:
Teacher of group or class is not used anymore, can we remove this ?

Reply:
Fine by me.